File: OnlineShopping/OnlineShopping/Models/MyOrderModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace OnlineShopping.Models
{
    public class MyOrderModel
    {
        public int OrderID { get; set; }
        public int UserID { get; set; }
        public double OrderTotal { get; set; }
        public List<CartModel> CartModel { get; set; }
    }
}

File: prjonlineshopping/src/app/models/MyOrder.model.ts
import { CartModel } from './Cart.model';

export class MyOrderModel {
    constructor() {
        this.OrderID = null;
        this.UserID = null;
        this.OrderTotal = 0;
        this.CartModel = null;
    }
    OrderID: number;
    UserID: number;
    OrderTotal: number;
    CartModel: CartModel[];
}
